src/scripts/page/conseils.ts

```diff
@@ -157,17 +156,17 @@ function showRelevantSuiviBlocks(element: HTMLElement, algoSuivi: AlgorithmeSuiv
         blockNames.push('conseils-sante')
         blockNames.push('conseils-sante-historique-symptomes')
 
-        const suiviRepetition: HTMLElement[] | null = Array.from(
-            element.querySelectorAll('.suivi-repetition')
+        const suiviRepetition = Array.from(
+            element.querySelectorAll<HTMLElement>('.suivi-repetition')
         )
         if (suiviRepetition) {
```

Comment:
Ce if n'est pas nécessaire car querySelectorAll retourne toujours une `NodeList` et le `forEach` suivant ne fera juste aucune itération.

Il y a plusieurs exemples similaires de `if` introduits à cause d'une type assertion fausse.